packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx

```diff
@@ -149,6 +149,29 @@ const Sidebar = (): JSX.Element => {
           mxPlanNode.routing.hash = '#/email_mxplan';
         }
 
+        /**
+         * Remove Identity Documents option
+         * Identity docments entry is added by default in ./navigation-tree/root.ts
+         */
+        let isIdentityDocumentsVisible;
+        if (results['identity-documents']) {
+          const { status } = await reketInstance.get(`/me/procedure/identity`);
+          if (!['required','open'].includes(status)) {
+            isIdentityDocumentsVisible = false;
+          }
+        } else {
+          isIdentityDocumentsVisible = false;
+        }
+        if (!isIdentityDocumentsVisible) {
```

Comment:
There is no case where `isIdentityDocumentsVisible = true`. Mistake ?

Reply:
Hi David, By default we have added the identity document option into sidebar and we are removing it when ever we have isIdentityDocumentsVisible has false/undefined. If it is true we wont remove it. Hence there is not action for True case.